ci/driver.sh

```diff
@@ -99,21 +114,43 @@ for pr in ${pr_list}; do
   echo "Processing Pull Request #${pr}"
   pr_dir="${GFS_CI_ROOT}/PR/${pr}"
   mkdir -p "${pr_dir}"
-  # call run_ci to clone and build PR
-  id=$(gh pr view "${pr}" --repo "${repo_url}" --json id --jq '.id')
-  "${pwd}/run_ci.sh" -p "${pr}" -d "${pr_dir}" -o "${pr_dir}/output_${id}"
+  # call clone-build_ci to clone and build PR
+  id=$("${GH}" pr view "${pr}" --repo "${repo_url}" --json id --jq '.id')
+  "${script_pwd}/clone-build_ci.sh" -p "${pr}" -d "${pr_dir}" -o "${pr_dir}/output_${id}"
   ci_status=$?
-  "${GH}" pr comment "${pr}" --repo "${repo_url}" --body-file "${GFS_CI_ROOT}/PR/${pr}/output_${id}"
   if [[ ${ci_status} -eq 0 ]]; then
-    "${GH}" pr edit --repo "${repo_url}" "${pr}" --remove-label "${CI_HOST}-Running" --add-label "${CI_HOST}-Passed"
+    #setup runtime env for correct python install
+    module use "${pr_dir}"/global-workflow/modulefiles
+    module load "module_base.${TARGET}"
+    cd "${pr_dir}/global-workflow/ush/python/pygw"
+    python3 -m pip install .
```

Comment:
Is it necessary to pip install?

Reply:
Looks like you are correct, there was no need to run pip install, but I did have to set **PYHTONPATH** to pick up the Host class in the workflow dir, maybe that should be in `ush/python/pygw` ?

Reply:
Talk to @aerorahul. He has been coordinating pythonpath stuff for the new UFSDA material.

Reply:
As you know, It turns out that I still needed it to pick up YAML from **setup.cnf** since it was not in HPC-Stack.  Thanks for suggesting the same fix I was thinking (on Monday).